automated install/basic-install.sh

```diff
@@ -187,6 +187,37 @@ if (echo > /dev/tcp/127.0.0.1/53) >/dev/null 2>&1; then
     echo -e "${OVER}  ${TICK} ${COL_LIGHT_GREEN}Port 53 is not in use, proceeding with install${COL_NC}"
   fi
 }
+
+#creating silent port 53 check
+silent53check(){
+# Print info regarding probing for open port 53
```

Comment:
This comment seems out of place since nothing prints in this function.

Reply:
Corrected.